apps/time/src/widgets/time-table/ui/TimeTableFilter.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import { ReactNode, useCallback } from 'react';
+
+import { cn } from '@clab/utils';
+
+import {
+  DAY_STATUS,
+  DayStatus,
+  useTimeTableParams,
+} from '@/widgets/time-table';
+
+interface TimeTableFilterItemProps {
+  children: ReactNode;
+  onClick?: () => void;
+  selected: boolean;
+}
+
+function TimeTableFilterItem({
+  children,
+  onClick,
+  selected,
+}: TimeTableFilterItemProps) {
+  return (
+    <button
+      className={cn(
+        'rounded-full px-4 py-2 transition-colors ',
+        selected ? 'bg-gray-200' : 'hover:bg-gray-100',
+      )}
+      type="button"
+      onClick={onClick}
+    >
+      {children}
+    </button>
+  );
+}
+
+export default function TimeTableFilter() {
+  const { dayStatus, searchParamsAction } = useTimeTableParams();
+
+  const onClickHandler = useCallback(
```

Comment:
```suggestion
  const handleFilterItemClick= useCallback(
```

Core팀에서 이벤트 함수에 대해 접두사를 `handle`, 접미사를 이벤트 명 `e.g. Change, Click`으로 사용하고 있어요, 해당 관련하여 팀 컨벤션을 문서화하여 공유할게요